JitsiMediaDevices.ts

```diff
@@ -107,11 +110,11 @@ class JitsiMediaDevices extends Listenable {
      * @returns {boolean} - True for granted and false for denied.
      * @throws {TypeError}
      */
-    _parsePermissionState(permissionStatus = {}) {
+    _parsePermissionState(permissionStatus: PermissionStatus = {} as PermissionStatus): boolean {
         // The status attribute is deprecated, and state
         // should be used instead, but check both for now
         // for backwards compatibility.
-        const status = permissionStatus.state || permissionStatus.status;
+        const status = permissionStatus.state || (permissionStatus as any).status;
```

Comment:
This likely no longer applies, can you check when the behavior was changed in the spec?